30 paskaita (egzaminas)/Task 2/script.js

```diff
@@ -3,3 +3,11 @@ Parašykite JS kodą, kuris skaičiuos kiek kartų buvo paspaustas mygtukas
 su tekstu "CLICK ME". Paspaudimų rezultatas turi būti matomas dešinėje
 pusėje esančiame "state" skaičiavimo bloke (<div id="btn__state">0</div>)
 ------------------------------------------------------------------- */
+
+
+let counter = 0;
+
+document.getElementById("btn__element").addEventListener('click', () => {
```

Comment:
Kad nekartoti ir neieskoti to paties elemento, galima issisaugoti `document.getElementById("btn__element")` grazinta reiksme i kintamaji.



```suggestion
const clickMeButton = document.getElementById("btn__element");

clickMeButton.addEventListener('click', () => {
```